### PR TITLE
W-12204799-Add-Kubernetes-service-note-CP

### PR DESCRIPTION
**Pull request by Crispy-Salesforce**
# Writer's Quality Checklist

Before merging your PR, did you:

- [ ] Run spell checker
- [ ] Run link checker to check for broken xrefs
- [ ] Check for orphan files
- [ ] Perform a local build and do a final visual check of your content, including checking for:
  - Broken images
  - Dead links
  - Correct rendering of partials if they are used in your content
  - Formatting issues, such as:
    - Misnumbered ordered lists (steps) or incorrectly nested unordered lists
    - Messed up tables
    - Proper indentation
    - Correct header levels
- [ ] Receive final review and signoff from:
  - Technical SME
  - Product Manager
  - Editor or peer reviewer
  - Reporter, if this content is in response to a reported issue (internal or external feedback)
- [ ] If applicable, verify that the software actually got released

### CODE REVIEW

File: gateway/1.5/modules/ROOT/pages/_partials/kubernetes-service-note.adoc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream service or you apply a policy that makes requests to a Kubernetes Service, you must set the following parameters in the `service` resource:
```
Comment (dmerlob) [inline gateway/1.5/modules/ROOT/pages/_partials/kubernetes-service-note.adoc]:
```suggestion
If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream service or you apply a policy that makes requests to a Kubernetes Service, set the following parameters in the `service` resource:
```
If the modal is not required, please omit it. See [modal verbs](https://confluence.internal.salesforce.com/display/MTDT/MuleSoft+CX+Writing+Style+Reference#:~:text=modal%20verbs.-,modal%20verbs,-Although%20some%20specific)

File: gateway/1.6/modules/ROOT/pages/_partials/kubernetes-service-note.adoc [hunk 2]
```diff
@@ -0,0 +1,10 @@
+If you use https://kubernetes.io/docs/concepts/services-networking/service/[Kubernetes Service^] as an upstream service or you apply a policy that makes requests to a Kubernetes Service, you must set the following parameters in the `service` resource:
```
Comment (dmerlob) [inline gateway/1.6/modules/ROOT/pages/_partials/kubernetes-service-note.adoc]: Same here. Omit the `you must` if you can.

**Review by dmerlob — DISMISSED** [review]

File: gateway/1.5/modules/ROOT/pages/_partials/kubernetes-service-note.adoc (hunk 1 above)
Comment (Crispy-Salesforce) [reply gateway/1.5/modules/ROOT/pages/_partials/kubernetes-service-note.adoc]: OK! Thanks!

File: gateway/1.6/modules/ROOT/pages/_partials/kubernetes-service-note.adoc (hunk 2 above)
Comment (Crispy-Salesforce) [reply gateway/1.6/modules/ROOT/pages/_partials/kubernetes-service-note.adoc]: OK! Thanks!

**Review by dmerlob — APPROVED** [review]

**Pull request merged**